sqlx-core/src/any/row.rs

```diff
@@ -82,6 +82,7 @@ impl AnyRow {
     pub fn map_from<'a, R: Row>(
         row: &'a R,
         column_names: Arc<crate::HashMap<UStr, usize>>,
+        is_sqlite: bool,
```

Comment:
I know this solution is far from elegant, but I don't understand `sqlx`'s architecture well enough to come up with a better solution.

Reply:
Yeah, this isn't the right place to fix this, but I see what needs to be done.